engine/engine/content/builtins/manifests/web/dark_theme.css

```diff
@@ -13,6 +13,7 @@
 		background-color: #1a72eb;
 		text-align: center;
 		line-height: 20px;
+		transition: width 1s ease;
```

Comment:
For smoother progress bar width update.

Reply:
I'm not sure from what version this particular feature was added, but not only for that feature, but for the PR in general it would be nice to check if it still works in Chrome 84 (I know that current implementation works fine there, just wanna make sure we don't use anything too modern which can reduce count of players)

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/CSS/transition#browser_compatibility it should work.

Reply:
This is a minor issue, but `width` is not a CSS animation property that can be GPU-accelerated. Also, animating `width` causes the element to be repainted every frame. It might not animate smoothly on under-powered devices.

See: [How to create high-performance CSS animations](https://web.dev/articles/animations-guide)

Animating `transform` instead of `width` could avoid both those problems. For example:

Start the progress bar with `transform-origin: left` and `transform: scaleX(0)` which will look similar to 0% width, then gradually expand it to `transform: scaleX(1)` via the `updateProgress` function, which will look similar to 100% width.

Reply:
Thanks for advice. I reworked progress animation with `transform` property instead of direct changes of `width` property.